recipes/atlas/build.sh

```
@@ -1,14 +1,9 @@
 #!/bin/bash
 
-export CFLAGS="$CFLAGS -I$PREFIX/include"
-export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
-export LD_LIBRARY_PATH="${PREFIX}/lib"
-export LIBRARY_PATH="${PREFIX}/lib"
-export CPATH="${PREFIX}/include"
+mkdir -p build
+cd build
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)